head_ref/src/head_ref.cpp

```diff
@@ -1,7 +1,9 @@
 #include "head_ref/HeadReference.h"
 
-int main(int argc, char** argv){
+#include <ros/ros.h>
```

Comment:
Question: what does this include do?

Reply:
Include everything from Ros...

Reply:
But why wasnt that needed before

Reply:
It was, but were inherited from other includes. That works, but isn't correct.